src/shared_utilities/drivers.jl

```diff
@@ -264,24 +266,27 @@ function turbulent_fluxes(
     u_air = p.drivers.u
     h_air = atmos.h
 
-    return turbulent_fluxes_at_a_point.(
-        T_sfc,
-        q_sfc,
-        ρ_sfc,
-        β_sfc,
-        h_sfc,
-        r_sfc,
-        d_sfc,
-        p.drivers.thermal_state,
-        u_air,
-        h_air,
-        atmos.gustiness,
-        model.parameters.z_0m,
-        model.parameters.z_0b,
-        Ref(model.parameters.earth_param_set),
-    )
+    dest .=
```

Comment:
We may not need the Ref(), because of [this](https://github.com/CliMA/ClimaLand.jl/blob/cd785c8f3c5cf984d2663bf1101db791c6e2971f/src/shared_utilities/Parameters.jl#L31)  in which case we could do @. dest = turb(). That might be a very marginal improvement?